batch/extension-jberet/src/main/java/org/wildfly/extension/batch/jberet/deployment/JobOperationUpdateStepHandler.java

```diff
@@ -34,6 +34,14 @@
  * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
  */
 abstract class JobOperationUpdateStepHandler extends JobOperationStepHandler {
```

Comment:
Ping me before doing about this as I may merge this soon for 10.1, in which case a rename can come later. It's not necessary to fix the bug which is the first goal.